filebrowser/tests/test_rock.py

```diff
@@ -45,7 +45,7 @@ def test_rock(rock_test_env):
             "run",
             "--entrypoint",
             "/bin/bash",
-            LOCAL_ROCK_IMAGE,
```

Comment:
I dont think we should change the env variables. same comment for changes bellow :)